content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -1,201 +1,206 @@
 
     ```
     # The following parameters can be customized based on your needs.
 
     CLUSTER_NAME="kubeflow-pipelines-standalone"
     ZONE="us-central1-a"
     MACHINE_TYPE="n1-standard-2" # A machine with 2 CPUs and 7.50GB memory
-    SCOPES="cloud-platform" # These scopes are needed for running some pipeline samples.
+    SCOPES="cloud-platform" # These scopes are needed for running some pipeline samples
 
     gcloud container clusters create $CLUSTER_NAME \
         --zone $ZONE \
         --machine-type $MACHINE_TYPE \
         --scopes $SCOPES
     ```
-     **WARNING**: Using `SCOPES="cloud-platform"` overgrants all GCP permissions to the cluster, so it's convenient to use. For a more secure cluster setup, refer to [Authenticating Pipelines to GCP](/docs/gke/authentication-pipelines/).
 
-    Reference:
+     **Warning**: Using `SCOPES="cloud-platform"` grants all GCP permissions to the cluster. For a more secure cluster setup, refer to [Authenticating Pipelines to GCP](/docs/gke/authentication/#authentication-from-kubeflow-pipelines).
 
-    - You can find available zones in https://cloud.google.com/compute/docs/regions-zones/#available.
-    - Get gcloud CLI tool at https://cloud.google.com/sdk/gcloud/.
-    - Read `gcloud container clusters create` command documentation at https://cloud.google.com/sdk/gcloud/reference/container/clusters/create.
+     **References**:
 
-1. Configure kubectl to talk to your newly created cluster. Refer to [Configuring cluster access for kubectl](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl).
+      * [GCP regions and zones documentation](https://cloud.google.com/compute/docs/regions-zones/#available)
 
-1. Deploy the latest version of Kubeflow Pipelines standalone to your cluster:
+      * [gcloud command-line tool guide](https://cloud.google.com/sdk/gcloud/)
 
-    ```
-    export PIPELINE_VERSION={{% pipelines/latest-version %}}
-    kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/base/crds?ref=$PIPELINE_VERSION
-    kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
-    kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
-    ```
+      * [gcloud command reference](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create)
 
-    Kubeflow Pipelines applications take a while (~3 minutes) to start.
-    
-    **Note**: The above approach is introduced in Kubeflow Pipelines version 0.2.0. For older versions please run the following instead:
-    ```
-    export PIPELINE_VERSION={{% pipelines/latest-version %}}
-    kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
-    ```
+1. Configure kubectl to talk to your newly created cluster. See the Google Kubernetes Engine (GKE) guide to
```

Comment:
Users will always need to configure kubectl to talk to their cluster. I would recommend moving this to the deploying section or adding a separate before you begin section.

Reply:
Totally agree.